src/server/server.ts

```diff
@@ -324,6 +360,38 @@ function compileCommon(req: express.Request, res: express.Response, wrapWithRunn
         });
 }
 
+const parseXmlString = (xml) => {
+    return new Promise((resolve, reject) => {
+        xml2js.parseString(xml, (err, result) => {
+            if (err) {
+                reject(err);
+            } else {
+                resolve(result);
+            }
+        });
+    });
+};
+
+function getVersionNumber(): Promise<any> {
+     return new Promise((resolve, reject) => {
+        return Request.post({
+            url: 'https://store.office.com/app/query?type=5&cmo=en-US&rt=xml',
```

Comment:
All string constants into top of file.  Both this URL and the Script Lab Store ID